SCons/Tool/ninja/__init__.py

```diff
@@ -421,9 +420,9 @@ def ninja_execute(self):
         target = self.targets[0]
         if target.get_env().get('NINJA_SKIP'):
             return
-        if target.check_attributes('ninja_file') is None or not target.is_conftest:
```

Comment:
The test I added revealed this was not needed because the SConf has its own isolated Taskmaster and task